apphub/image_generation/pggan_nihchestxray/pggan_nihchestxray.py

```diff
@@ -154,62 +155,79 @@ def on_epoch_end(self, state):
                 else:
                     plt.imshow(disp_img)
                 plt.axis('off')
-            plt.savefig(os.path.join(self.save_dir, 'image_at_{:08d}.png').format(state["epoch"]))
+                disp_img = np.uint8(disp_img * 255)
+                cv2.imwrite(os.path.join(self.save_dir, 'image_at_{:08d}_{}.png').format(state["epoch"], i), disp_img)
```

Comment:
why are we writing this twice?

Reply:
cv2 is saving pure image,   plt is saving a grid of images.